examples/sampling/pyg/test.py

```diff
@@ -0,0 +1,191 @@
+import dgl.graphbolt as gb
+import torch
+import torch.nn.functional as F
+from torch_geometric.data import Data
+from torch_geometric.loader import NeighborSampler
+from torch_geometric.nn import SAGEConv
+
+
```

Comment:
Add docstring at the top. please take https://github.com/dmlc/dgl/blob/master/examples/sampling/node_classification.py as a good reference.

Reply:
ok

Reply:
Thx! Working on it!